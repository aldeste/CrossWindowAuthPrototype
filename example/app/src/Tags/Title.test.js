import Title from "./Title";
import renderer from "react-test-renderer";
import React from "react";
import "jest-styled-components";

describe("Title tag", () => {
  it("Should be defined", () => {
    expect(Title).toBeDefined();
  });

  it("Is a styled component", () => {
    expect(Title.name).toBe("StyledComponent");
  });

  it("Matches earlier configuration", () => {
    const tree = renderer.create(<Title />).toJSON();
    expect(tree).toMatchStyledComponentsSnapshot();
  });
});
