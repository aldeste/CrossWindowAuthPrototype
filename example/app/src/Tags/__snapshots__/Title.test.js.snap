// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title tag Matches earlier configuration 1`] = `
.cXYtHj {
  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
  font-size: 2.824em;
  font-weight: 500;
  letter-spacing: 0.006em;
  line-height: 1.04167;
}

<h1
  className="Title__Text-hPdHUJ cXYtHj"
/>
`;
